webpack.mix.js

```diff
@@ -15,11 +15,15 @@ const config = {
     ],
 };
 
+const cssUrls = {
+    processCssUrls: false
+}
+
 Mix.options(options).webpackConfig(config)
    .js('assets/source/js/admin.js', 'assets/dist/js')
    .js('assets/source/js/editor.js', 'assets/dist/js')
    .js('assets/source/js/theme.js', 'assets/dist/js')
-   .sass('assets/source/sass/admin.scss', 'assets/dist/css')
-   .sass('assets/source/sass/editor.scss', 'assets/dist/css')
-   .sass('assets/source/sass/theme.scss', 'assets/dist/css')
+   .sass('assets/source/sass/admin.scss', 'assets/dist/css').options(cssUrls)
+   .sass('assets/source/sass/editor.scss', 'assets/dist/css').options(cssUrls)
+   .sass('assets/source/sass/theme.scss', 'assets/dist/css').options(cssUrls)
```

Comment:
Maybe the processCssUrls option should be moved to the options object instead?

As far as I know, options is a global thing is Laravel Mix.

Reply:
True! Works in the options object aswell.